src/input_formatter/text_formatter.test.tsx

```diff
@@ -0,0 +1,259 @@
+import { emptyIssueList, listAllIssues } from "../issueHandling/issueMapping";
+import {
+	detectBracketErrors,
+	detectMissingStatementsOrConnector,
+	preFormatExpressionFromImput,
+	replaceExpressionElementsIntoPrologCode,
+} from "./expression_formatter";
+import { sentenceIntoWordList, textFormatter } from "./text_formatter";
+
+test("splitting a single sentences into List of words: ", () => {
+	emptyIssueList();
+	const result = sentenceIntoWordList("Hello, this is a test");
+	const expectedResult = ["Hello", "this", "is", "a", "test"];
+	expect(result).toEqual(expectedResult);
+});
```

Comment:
In the future you could use the describe-style for your tests. It makes what's tested more intuitive and your tests get more readable. This can for example look like this (taken from detectWrongSyntax.test.ts):

```
describe("logSingleWord", () => {
	it("Logs a ordinary word correctly", () => {
		const position: Position = {fromIndex: 0, toIndex: 4};
		const issues = logSingleWord("Hello", position);
		expect(issues).toEqual({
			code: issueJson.INVALID_WORD.message,
			message: `${"Hello"} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
			position: {
				fromIndex: position.fromIndex,
				toIndex: position.toIndex,
			},
			severity: issueJson.INVALID_WORD.severity,
		});
	});
```

You can even use multiple describes. Here's an example from diprocheInterface.test.ts:

```
describe("Error cases", () => {
		describe("Predicate is invalid", () => {
			it("Adds none of the regular predicates", () => {
				const mode = "Just a random text" as Mode;
				expect(addPredicate("", mode)).toEqual("Es gibt kein passendes Prädikat.");
			});
		});
	});
```

That is especially useful to distinguish different kinds of testcases.